### PR TITLE
Ogc 183 staatskalender

### PR DESCRIPTION
**Pull request by BreathingFlesh**

### CODE REVIEW

File: src/onegov/agency/templates/macros.pt [hunk 1]
```diff
@@ -263,9 +263,11 @@
                 <a href="${request.link(agency)}" i18n:translate="">Show agency</a>
             </div>
             <ul class="memberships fa-ul" tal:condition="memberships">
-               <li tal:repeat="membership memberships" class="person"><span class="fa-li"><i class="fa fa-user-circle"></i></span>
-                   <b><a href="${request.link(membership.person)}">${membership.person.title}</a></b><tal:block tal:condition="membership.title">, <a href="${request.link(membership.person)}">${membership.title}</a></tal:block>
-               </li>
+                <tal:block tal:repeat="membership memberships" >
+                    <li tal:condition="(membership.access == 'public' and membership.published and membership.person.access == 'public' and membership.person.published) or request.is_logged_in" class="person"><span class="fa-li"><i class="fa fa-user-circle"></i></span>
```
Comment (msom) [inline src/onegov/agency/templates/macros.pt:267]: `membership.person` might be `None` in some edge case... It's probably better to check:
logged in OR (membership access AND membership public AND (not person OR (person access AND person public)))

File: tests/onegov/agency/test_views.py [hunk 2]
```diff
@@ -713,6 +717,42 @@ def test_basic_search(client_with_es):
...
+        client.get(url)
+
+    # Reindexing takes a few seconds
+    time.sleep(5)
+
+    # Test search results
+    with freeze_time(in_1_hour):
+        assert 'Rivera' in client.get('/search?q=Nick')
+        assert 'Nick' in client.get('/search?q=Rivera')
+
```
Comment (msom) [inline tests/onegov/agency/test_views.py:754]: Please extend the test to cover `publication_end` / unindexing as well

**Review by msom — CHANGES_REQUESTED** [review]
